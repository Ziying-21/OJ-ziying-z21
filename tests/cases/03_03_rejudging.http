###
# Send request:
PUT http://127.0.0.1:12345/jobs/0 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 404
# content-length: 63
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:12 GMT
# 
# {"code":3,"message":"Job 0 not found","reason":"ERR_NOT_FOUND"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 1137
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:12 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0},{"id":2,"info":"","memory":0,"result":"Waiting","time":0},{"id":3,"info":"","memory":0,"result":"Waiting","time":0},{"id":4,"info":"","memory":0,"result":"Waiting","time":0},{"id":5,"info":"","memory":0,"result":"Waiting","time":0},{"id":6,"info":"","memory":0,"result":"Waiting","time":0},{"id":7,"info":"","memory":0,"result":"Waiting","time":0},{"id":8,"info":"","memory":0,"result":"Waiting","time":0},{"id":9,"info":"","memory":0,"result":"Waiting","time":0},{"id":10,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:13.148Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 1222
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:13 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":162006},{"id":1,"info":"","memory":0,"result":"Accepted","time":63015},{"id":2,"info":"","memory":0,"result":"Accepted","time":1989},{"id":3,"info":"","memory":0,"result":"Accepted","time":1852},{"id":4,"info":"","memory":0,"result":"Accepted","time":1830},{"id":5,"info":"","memory":0,"result":"Accepted","time":1545},{"id":6,"info":"","memory":0,"result":"Accepted","time":1481},{"id":7,"info":"","memory":0,"result":"Accepted","time":1851},{"id":8,"info":"","memory":0,"result":"Accepted","time":1804},{"id":9,"info":"","memory":0,"result":"Accepted","time":1817},{"id":10,"info":"","memory":0,"result":"Accepted","time":1776}],"created_time":"2022-09-09T02:19:13.148Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-09T02:19:13.406Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 406
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:13 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:14.159Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 454
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:14 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":67106},{"id":1,"info":"","memory":0,"result":"Accepted","time":54396}],"created_time":"2022-09-09T02:19:14.159Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-09T02:19:14.286Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/0 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 200
# content-length: 1161
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:15 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0},{"id":2,"info":"","memory":0,"result":"Waiting","time":0},{"id":3,"info":"","memory":0,"result":"Waiting","time":0},{"id":4,"info":"","memory":0,"result":"Waiting","time":0},{"id":5,"info":"","memory":0,"result":"Waiting","time":0},{"id":6,"info":"","memory":0,"result":"Waiting","time":0},{"id":7,"info":"","memory":0,"result":"Waiting","time":0},{"id":8,"info":"","memory":0,"result":"Waiting","time":0},{"id":9,"info":"","memory":0,"result":"Waiting","time":0},{"id":10,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:13.148Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-09T02:19:13.406Z"}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 1223
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:16 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":163255},{"id":1,"info":"","memory":0,"result":"Accepted","time":306284},{"id":2,"info":"","memory":0,"result":"Accepted","time":1829},{"id":3,"info":"","memory":0,"result":"Accepted","time":1771},{"id":4,"info":"","memory":0,"result":"Accepted","time":1599},{"id":5,"info":"","memory":0,"result":"Accepted","time":1594},{"id":6,"info":"","memory":0,"result":"Accepted","time":1591},{"id":7,"info":"","memory":0,"result":"Accepted","time":1619},{"id":8,"info":"","memory":0,"result":"Accepted","time":1889},{"id":9,"info":"","memory":0,"result":"Accepted","time":1560},{"id":10,"info":"","memory":0,"result":"Accepted","time":1559}],"created_time":"2022-09-09T02:19:13.148Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-09T02:19:15.670Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/1 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 200
# content-length: 430
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:16 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:14.159Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-09T02:19:14.286Z"}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 454
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:17 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":64306},{"id":1,"info":"","memory":0,"result":"Accepted","time":54313}],"created_time":"2022-09-09T02:19:14.159Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-09T02:19:16.311Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/2 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 404
# content-length: 63
# access-control-expose-headers: content-type
# content-type: application/json
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# date: Fri, 09 Sep 2022 02:19:17 GMT
# 
# {"code":3,"message":"Job 2 not found","reason":"ERR_NOT_FOUND"}
