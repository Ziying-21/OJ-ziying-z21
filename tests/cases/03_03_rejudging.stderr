[2022-09-09T02:19:12Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:19:12Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:19:13Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000196
[2022-09-09T02:19:13Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/0 HTTP/1.1" 404 63 "-" "-" 0.000184
[2022-09-09T02:19:13Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 1137 "-" "-" 0.000524
[2022-09-09T02:19:14Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 1222 "-" "-" 0.000419
[2022-09-09T02:19:14Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 406 "-" "-" 0.000976
[2022-09-09T02:19:15Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 454 "-" "-" 0.000281
[2022-09-09T02:19:15Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/0 HTTP/1.1" 200 1161 "-" "-" 0.000912
[2022-09-09T02:19:16Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 1223 "-" "-" 0.000644
[2022-09-09T02:19:16Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/1 HTTP/1.1" 200 430 "-" "-" 0.001827
[2022-09-09T02:19:17Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 454 "-" "-" 0.000340
[2022-09-09T02:19:17Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/2 HTTP/1.1" 404 63 "-" "-" 0.000367
